lib/resources/platform.rb

```diff
@@ -65,6 +65,23 @@ def params
       h
     end
 
+    def detect(indent: 0, color: 39)
+      str = ''
```

Comment:
Should that be private? I am not sure if we want to expose this via our resource